src/components/cardDetail/index.tsx

```
@@ -0,0 +1,77 @@
+        </div>
+      </Dialog.Title>
+      <Dialog.Close toggleModal={toggleModal} />
+      <Dialog.Content className='mt-10 flex grow-0 flex-col-reverse justify-between tablet:mt-24 tablet:flex-row tablet:gap-13'>
+        <section className='tablet:w-420 pc:w-445'>
+          <div className='mt-16 flex tablet:mt-0'>
+            <div className='border-r-1 border-r-gray-300 pr-12 tablet:pr-20'>
+              <Badge>In Progresssss</Badge>
+            </div>
+            <div className='ml-12 flex h-fit flex-wrap items-center gap-8 tablet:ml-20'>
+              {tagList1.map((tag) => {
+                return <Tag className='bg-amber-100'>{tag}</Tag>;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)